src/Components/SignUpForm/SignUpForm.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import SignIn from '../../Containers/SignIn/SignIn'
+import SignUp from '../../Containers/SignUp/SignUp'
+
+const SignUpForm = () => {
+  return (
+    <div>
+      <SignIn />
```

Comment:
How are these going to be conditionally rendered?

Reply:
The signUpForm will have properties in state that change from true to false depending on whether the user clicks on the ‘log in’ or ‘sign up’ button

Reply:
Awesome. Thank you for the swift reply